src/daemon/rpmostree-sysroot-upgrader.cxx

```diff
@@ -517,9 +517,6 @@ rpmostree_sysroot_upgrader_pull_base (RpmOstreeSysrootUpgrader  *self,
             if (!ostree_repo_pull_with_options (self->repo, origin_remote, opts, progress,
                                                 cancellable, error))
               return glnx_prefix_error (error, "While pulling %s", override_commit ?: origin_ref);
-
-            if (progress)
-              ostree_async_progress_finish (progress);
```

Comment:
Hmm.  Shouldn't we include the `_finish()` call elsewhere though?  Is this related?

Reply:
We `_finish()` at a higher level. We need to be consistent about calling `_finish()` at the same scope level as `ostree_async_progress_new()` so it's clear they all match up. Have to split out that patch.